tests/Fuerte/test_10000_writes.cpp

```diff
@@ -0,0 +1,193 @@
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Christoph Uhde
+////////////////////////////////////////////////////////////////////////////////
+
+#include "test_main.h"
+#include <fuerte/fuerte.h>
+#include <fuerte/loop.h>
+#include <fuerte/helper.h>
+
+#include <velocypack/Parser.h>
+#include <velocypack/Iterator.h>
+#include <velocypack/velocypack-aliases.h>
+
+#include <chrono>
+#include <iostream>
+#include <fstream>
+#include <boost/thread.hpp>
+#include <boost/asio/io_service.hpp>
+#include <thread>
+
+namespace f = ::arangodb::fuerte;
+
+class Connection100kWritesF : public ::testing::Test {
+ protected:
+  Connection100kWritesF(){
+    _server = "vst://127.0.0.1:8529";
+  }
+  virtual ~Connection100kWritesF() noexcept {}
+
+  virtual void SetUp() override {
+    try {
+      f::ConnectionBuilder cbuilder;
+      cbuilder.host(_server);
+      _connection = cbuilder.connect(_eventLoopService);
+
+      //delete collection
+      {
+        auto request = arangodb::fuerte::createRequest(
+          arangodb::fuerte::RestVerb::Delete, "/_api/collection/testobi");
+        auto result = _connection->sendRequest(std::move(request));
+        //arangodb::fuerte::run();
+      }
+      //create collection
+      {
+        namespace fu = arangodb::fuerte;
+        VPackBuilder builder;
+        builder.openObject();
+        builder.add("name" , VPackValue("testobi"));
+        builder.close();
+        arangodb::fuerte::Request request = *fu::createRequest(fu::RestVerb::Post, "/_api/collection");
+        request.addVPack(builder.slice());
+        auto result = _connection->sendRequest(std::move(request));
+        if (result->header.responseCode >= 400){
+          std::cerr << fu::to_string(request);
+          std::cerr << fu::to_string(*result);
+          ASSERT_TRUE(false);
+          throw;
+        }
+      }
+    } catch(std::exception const& ex) {
+      std::cout << "SETUP OF FIXTURE FAILED" << std::endl;
+      throw ex;
+    }
+  }
+
+  virtual void TearDown() override {
+    //delete collection
+    {
+      auto request = arangodb::fuerte::createRequest(
+        arangodb::fuerte::RestVerb::Delete, "/_api/collection/testobi");
+      auto result = _connection->sendRequest(std::move(request));
+      //arangodb::fuerte::run();
+    }
+    _connection.reset();
+  }
+
+  std::shared_ptr<f::Connection> _connection;
+
+ private:
+  f::EventLoopService _eventLoopService;
+  std::string _server;
+  std::string _port;
+};
+
+namespace fu = ::arangodb::fuerte;
+
+/*
+TEST_F(Connection100kWritesF, Writes100k){
```

Comment:
Why is this test disabled?

Reply:
This test file does not build properly anymore, but could possibly deleted